msdsales/errors_test.go

```diff
@@ -0,0 +1,91 @@
+package msdsales
+
+import (
+	"errors"
+	"net/http"
+	"testing"
+
+	"github.com/amp-labs/connectors/common"
+	"github.com/amp-labs/connectors/common/interpreter"
+)
+
+func Test_interpretJSONError(t *testing.T) {
```

Comment:
We generally name tests without underscores, so this would be: TestInterpretJSONError